.github/workflows/release.yaml

```diff
@@ -13,8 +13,33 @@ env:
   SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
   SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
 
+concurrency:
+   group: release-${{ github.ref }}
+   cancel-in-progress: true
+
 jobs:
-  dist:
+  atomic-release:
+    # This creates a node in the DAG so that all of the "push" tasks
+    # can depend on all build tasks.
+    name: Atomic Release Release
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags/v')
+    needs:
+      - python-linux
+      - python-macos
+      - python-windows
+      - kernel-dist
+      - node-build
```

Comment:
question: haven't looked below yet, just sort of came across my mind - why does `python` need 3 separate jobs, but `kernel-dist` and `node-build` don't?

Reply:
just how it's factored. the kernel and node are also 3 jobs, but they are matrixes and so we can use the group reference.

I think you could factor this as a single task, and get rid of the empty `atomic-release` task, but it'd be much harder to follow, and this was a simpler migration. These are pretty hard configurations to test (and are pretty expensive,) so I went for minimal impact.